features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -187,11 +189,15 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
     const [ defaultActiveIndex, setDefaultActiveIndex ] = useState<number>(undefined);
     const [ totalTabs, setTotalTabs ] = useState<number>(undefined);
     const [ isM2MApplication, setM2MApplication ] = useState<boolean>(false);
+    const { legacyAuthzRuntime } = useAuthorization();
 
     const eventPublisher: EventPublisher = EventPublisher.getInstance();
 
     const isFragmentApp: boolean = application?.advancedConfigurations?.fragment || false;
     const hiddenAuthenticators: string[] = [ ...(UIConfig?.hiddenAuthenticators ?? []) ];
+    const myAccountSimplifiedSettings: boolean = !useSelector((state: AppState) =>
+        state.config.ui.features.applications?.disabledFeatures.includes("applications.myaccount.simplifiedSettings"));
```

Comment:
Let's use the `useSelector` hook only for retrieving values.

```suggestion
    const disabledApplicationFeatures: string[] = useSelector((state: AppState) =>
        state.config.ui.features.applications?.disabledFeatures);
```